src/main/java/seedu/duke/model/template/BooleanTemplate.java

```diff
@@ -1,5 +1,6 @@
 package seedu.duke.model.template;
 
+import seedu.duke.Duke;
```

Comment:
Why is this needed?

Reply:
Accidently got added cuz intellij optimised the imports lol